genai-perf/tests/test_config_command.py

```diff
@@ -267,6 +284,45 @@ def test_tokenizer(self):
         self.assertEqual(config.tokenizer.revision, "test_revision")
         self.assertEqual(config.tokenizer.trust_remote_code, True)
 
+    def test_deepcopy_config(self):
+        """
+        Test that the configuration can be deepcopied
+        """
+        # yapf: disable
+        yaml_str = ("""
+            model_name: gpt2
```

Comment:
why do you list model names as `model_name: gpt2` here but above in `test_multi_model_name` you have a list with each item on a separate line? Do they both work the same way?